lib/krane/restart_task.rb

```diff
@@ -49,18 +49,18 @@ def run(*args)
 
     # Runs the task, raising exceptions in case of issues
     #
-    # @param deployments_names [Array<String>] Array of workload names to restart
+    # @param deployments [Array<String>] Array of workload names to restart
     # @param selector [Hash] Selector(s) parsed by Krane::LabelSelector
     # @param verify_result [Boolean] Wait for completion and verify success
     #
     # @return [nil]
-    def run!(deployments_names = nil, selector: nil, verify_result: true)
+    def run!(deployments: nil, selector: nil, verify_result: true)
```

Comment:
I think deployments is the right name here. There was talk about supporting DS's as well, under a new keyword. I think that's better than a generic `resources` where we have to manually fetch both and check to see what exists.